pkg/service/account/handler_account.go

```diff
@@ -586,15 +588,27 @@ func (p *Server) CreateUser(ctx context.Context, req *pb.CreateUserRequest) (*pb
 				Addresses:   []string{email},
 			})
 		} else {
+			platformName := pi.Global().GetBasicCfg().PlatformName
+			platformUrl := pi.Global().GetBasicCfg().PlatformUrl
+
+			accountClient, err := accountclient.NewClient()
```

Comment:
Here is account manager, should use imClient, accountclient is used for other manager.